### ISSUE TITLE
Could you separate tp as a independent flag?

### ISSUE THREAD

**Issue by 5igm** [bekvon/Residence#121]
There is an option in flags file.

```
  ResidenceDefault:
    tp: false
```

I hope this option can be available in CreatorDefault as well.
Now, if I try to disable this option for creator, teleportation from other plugins will also be disabled.

Thank you very much.

**Comment by Zrips** [comment]
Check config file and there is section BlockAnyTeleportation where you can disable it and then you can teleport to residences with 3rd party plugins. This was implemented some time ago to prevent player teleportation by setting home in that residence or by allowing other players to teleport to you when they don't have teleport access. Currently its more or less deprecated but its there just for convenience.

**Comment by 5igm** [comment]
But BLockAnyTeleportation disabled the teleportation from 3rd party plugins like essentials as well.

**Comment by 5igm** [comment]
@Zrips

**Comment by Zrips** [comment]
That was its purpose, to block teleportation from 3rd party plugins, if you want to allow it, then just disable this feature

**Comment by 5igm** [comment]
Yes, I want to disable this feature, but I don't know how to do it. I don't want players using "/res rt" and "/res tp", yet I want they can use "/tpa", "/back", etc. from 3rd plugins.
@Zrips

**Comment by 5igm** [comment]
If I set BlockAnyTeleportation as true, all teleportation will be disabled in residence area. For example, I installed authme in my server. Before users login into the server, they will be teleported to Spawn point. But this action was stopped by residence, because I used residence to protect the spawn point and I set BlockAnyTeleportation as true.
